css/workflow-export-dialog.styl

```diff
@@ -0,0 +1,11 @@
+.workflow-export-list
+  padding-left: 0
+
+  li
```

Comment:
Let's actually target these with a class rather than the tag. We've been working toward using a BEM convention for class names.